projects/storefrontlib/shared/components/form/form-errors/form-errors.component.ts

```diff
@@ -34,6 +35,8 @@ import { map, startWith } from 'rxjs/operators';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class FormErrorsComponent implements DoCheck {
+  private featureConfigService = inject(FeatureConfigService);
```

Comment:
I think this should be a protected Optional() injection.

Reply:
We have documented here https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-UsingFeatureTogglesinTypeScript (2.18.5.1. Using Feature Toggles in TypeScript:) to use `private` access modifier for feature config service.

![image](https://github.com/SAP/spartacus/assets/129765378/56980663-de28-4ba0-a147-a5a2ec9acf3c)

If to talk about Optional(), I don't see the reason to use it. I think it is not the same case as with constructor. We are declaring this service as a new private property of the class. So it always will be there. Please, let me know if you see any cases for usage of the optional decorator.

Reply:
Ah I didn't know about this. Thanks!